.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -15,28 +15,46 @@ This link describes the WinUI feature/API proposal process:
 https://github.com/Microsoft/microsoft-ui-xaml/blob/master/docs/feature_proposal_process.md
 -->
 
-<!-- Add a title for your feature or API proposal. Please be short and descriptive -->
 # Proposal: [your title here] 
+<!-- Add a title for your feature or API proposal. Please be short and descriptive -->
 
 ## Summary
-<!-- Please include a couple sentences summarizing your feature or API proposal -->
+<!-- Include 1-2 sentences summarizing your feature or API proposal -->
 
 ## Rationale
-<!-- Please describe WHY the feature should be added to WinUI for all developers and users.
-If applicable you can also describe how it aligns to the current WinUI roadmap and priorities:
+<!-- Fill in the table below to describe WHY the feature should be added to WinUI for all developers and users.
+Proposals often have multiple motives for why we should do the work, so list each one as a separate row item.
+If applicable you can also describe how it aligns to the current WinUI roadmap and priorities in a separate paragraph:
 https://github.com/Microsoft/microsoft-ui-xaml/blob/master/docs/roadmap.md
 -->
+| Motive |
+| :----- |
+| {list each motivating reason as a separate row} |
+
 
 <!----------------------
 The below sections are optional when submitting an idea or proposal.
 All sections are required before we'll accept a PR to master, but aren't necessary to start the discussion.
 ------------------------>
 
-## Functional Requirements
-<!-- Please include a prioritized list of what the feature should and shouldn't do -->
+## Scope
+<!-- Please include a list of what the feature should and shouldn't do by filling in the table below.
+'Must' implies that the feature should not ship without this capability.  
+'Should' is something we should push hard for, but is not absolutely required to ship.
+'Could' is a nice-to-have; a good stretch goal that isn't painful if we don't achieve it.
+'Won't' is a clear statement that the proposal/feature will intentionally not have that capability.
+This list will evolve and grow as the proposal becomes more refined over time.
+A good rule of thumb is to start your proposal with no more than 7 high-level requirements.
+-->
+| Capability | Priority |
+| :---------- | :------- |
+| Feature can do W | Must |
```

Comment:
Should these be feature focused (ie. "New tabs can be added") or user focused (ie. "User can add new tabs")? I generally think that user focused scenarios are more concise and help demonstrate the value of a particular feature.

Reply:
I like your suggestion, but I don't like "User can blah" since "User" is an ambiguous term that sometimes means End User and sometimes Developer.  I could turn these into "developers can do X" rather than "feature can do X" -- would that maintain the spirit of what you're saying?

Reply:
Yup, that'd work. The only thing I'd consider is exactly what you pointed out - sometimes the "user" is the End User and sometimes the Developer. Some features are built for one (ie. "Developer can bind to a collection of tabs") and some are for the other (ie. "End users observe great tab reordering animations"). I chose "User" because it was intentionally ambiguous - but it also focuses on the person being impacted -- not the feature itself.

Reply:
I just updated the PR with a more verbose "This proposal will allow developers to accomplish W" phraseology.  People can also adjust this to talk about end users rather than developers (if that's appropriate), and the example I put into the template makes that clear.